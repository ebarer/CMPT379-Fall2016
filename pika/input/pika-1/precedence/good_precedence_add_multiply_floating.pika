exec {

  print 4.8 + 3.4 * 1.5, _n_.
  print 2.3 + 7.4 * 1.0, _n_.
  print 2.6 + 6.8 * 4.0, _n_.
  print 4.5 + 3.4 * 2.8, _n_.
  print 4.7 + 5.8 * 8.3, _n_.
  print 1.2 + 8.7 * 1.8, _n_.
  print 4.5 + 9.3 * 4.4, _n_.
  print 5.8 + 9.6 * 3.9, _n_.
  print 1.8 + 3.4 * 3.3, _n_.
  print 7.5 + 8.0 * 3.4, _n_.
  print 6.8 + 8.7 * 4.1, _n_.
  print 5.1 + 3.6 * 9.0, _n_.

  print 9.6 * 5.7 + 9.0, _n_.
  print 2.0 * 7.8 + 5.7, _n_.
  print 5.1 * 1.7 + 4.9, _n_.
  print 3.9 * 9.8 + 7.1, _n_.
  print 2.8 * 7.2 + 9.6, _n_.
  print 4.7 * 9.9 + 5.6, _n_.
  print 4.3 * 8.7 + 2.5, _n_.
  print 7.5 * 7.8 + 6.5, _n_.
  print 7.1 * 1.4 + 7.7, _n_.
  print 7.3 * 9.1 + 9.8, _n_.
  print 7.7 * 5.0 + 4.5, _n_.
  print 2.2 * 9.8 + 7.3, _n_.

  print 6.5 + 8.0 * 8.4 + 2.3, _n_.
  print 8.1 + 7.3 * 2.5 + 2.9, _n_.
  print 9.1 + 7.2 * 6.8 + 1.7, _n_.
  print 5.3 + 8.6 * 1.3 + 1.7, _n_.
  print 3.5 + 4.5 * 5.8 + 7.8, _n_.
  print 6.1 + 1.4 * 8.8 + 7.5, _n_.
  print 2.2 + 6.1 * 9.2 + 9.3, _n_.
  print 4.5 + 4.5 * 3.7 + 9.1, _n_.
  print 9.9 + 3.4 * 8.5 + 8.6, _n_.
  print 3.7 + 4.6 * 4.6 + 2.7, _n_.
  print 4.8 + 3.8 * 5.5 + 5.0, _n_.
  print 3.0 + 5.9 * 7.5 + 7.5, _n_.

  print 9.4 * 2.8 + 3.9 * 6.4, _n_.
  print 2.9 * 7.7 + 6.3 * 2.5, _n_.
  print 6.4 * 5.9 + 1.7 * 6.3, _n_.
  print 1.7 * 1.9 + 4.0 * 7.9, _n_.
  print 6.7 * 9.0 + 7.0 * 3.1, _n_.
  print 7.1 * 5.9 + 7.7 * 6.3, _n_.
  print 9.1 * 8.1 + 5.8 * 1.0, _n_.
  print 8.6 * 7.5 + 6.1 * 5.3, _n_.
  print 5.6 * 8.5 + 1.3 * 5.9, _n_.
  print 6.1 * 7.1 + 8.4 * 3.9, _n_.
  print 9.4 * 6.4 + 1.3 * 6.9, _n_.
  print 4.2 * 2.0 + 8.1 * 4.4, _n_.

}

