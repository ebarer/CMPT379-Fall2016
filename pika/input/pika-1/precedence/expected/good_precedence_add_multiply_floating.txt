9.9
9.7
29.8
14.02
52.84
16.86
45.42
43.24
13.02
34.7
42.47
37.5
63.72
21.3
13.57
45.32
29.76
52.13
39.91
65
17.64
76.23
43
28.86
76
29.25
59.76
18.18
37.4
25.92
67.62
30.25
47.4
27.56
30.7
54.75
51.28
38.08
48.47
34.83
82
90.4
79.51
96.83
55.27
76.07
69.13
44.04
